jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/comments/CaseCommentsPresenter.java

```diff
@@ -138,7 +137,10 @@ protected void updateCaseComment(final CaseCommentSummary caseCommentSummary,
 
     protected void deleteCaseComment(final CaseCommentSummary caseCommentSummary) {
         caseService.call(
-                (Void) -> refreshComments()
+            (Void) -> {
+                visibleCaseComments.remove(caseCommentSummary);
+                refreshComments();
```

Comment:
I think you can avoid calling `refreshComments` as you already removed the item from the list. You only need to update the view now.